### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by staudamm**

### CODE REVIEW

File: src/PlayObjects.py [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+            if self.slot == 25:
+                self.client.trigger_address('/trig_master', 25)
+                self.play = False
+                print('self.play? {} alive?: {}'.format(self.play, self.is_alive()))
+                return
+
+
+        print('alive?: ', self.is_alive())
+
+
```
Comment (staudamm) [inline src/PlayObjects.py]: This function is weired: it does not return anything. What does it do?

File: src/TextEdit.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                self.feedback.controller_send()
+                self.linenr = 0
+
+    def new_beitrag(self, user, intent, ref_text, user_text, eigencat):
+        self.saetze = re.split(r'[\r\n]+', user_text)  # jede Zeile = Satz
+        # self.feedback.init_send(user)  # falls ich per button den Song starten will
+
+        if len(self.saetze) <= 1:
+            self.one_satz(user, intent, ref_text, self.saetze[0], eigencat)
+
```
Comment (staudamm) [inline src/TextEdit.py]: remove blank line

File: src/Trainer.py [hunk 3]
```diff
@@ -1,3 +1,7 @@
+'''
+
+
```
Comment (staudamm) [inline src/Trainer.py]: I guess this block can be removed?

File: src/UDPClient.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+from pythonosc.udp_client import SimpleUDPClient
+from itertools import *
+import time
+from datetime import datetime
+
+DATETIME = datetime(2019, 6, 20, 20, 15, 30, 11111)
+START = time.mktime(DATETIME.timetuple()) + DATETIME.microsecond / 1E6
+
+class ClientIO(SimpleUDPClient):
+
```
Comment (staudamm) [inline src/UDPClient.py]: remove blank line

File: src/UDPClient.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    def send_controlmap(self, map):
+        self.send_message("/controlmap", map)
+
+    def send_start(self, msg):
+        self.send_message("/start", msg)
+
+
+class Client_MusicServer(SimpleUDPClient):
+
```
Comment (staudamm) [inline src/UDPClient.py]: remove blank line

File: src/UDPClient.py [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+    def tempo_coarse(self, coarse):
+        print('coarse: {}'.format(int(coarse)))
+        self.send_message("/tempo_coarse", int(coarse))
+
+    def mute_send(self, channel, ccnr, val):
+        print('mute send: {}'.format(ccnr))
+        if val == 0:
+            self.send_message('{}_cc{}'.format(channel, ccnr), 0)
+            self.send_message('{}_{}'.format(channel, ccnr), 0)
+
```
Comment (staudamm) [inline src/UDPClient.py]: remove blank line

File: src/UDPClient.py [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+        else:
+            self.send_message('{}_cc{}'.format(channel, ccnr), 127)
+
+        #self.send_message('{}_{}'.format(channel, ccnr), val)
```
Comment (staudamm) [inline src/UDPClient.py]: I'm sure this line can be deleted

File: src/UserClassifier03.py [hunk 8]
```diff
@@ -1,8 +1,5 @@
 """
-perspektivisch: 1) pro kategorie ein Track/Synthsizer?
-2) bei neuen Texten: a) in UserOrdner boris/Zustimmung.txt b) alle_user/Zustimmung.txt
-vor allem die b) lösung wäre eine wachsende Datenbank, die zusätzlich zu meiner bisherigen TrainingData
-(die ja letztlich eine Boris-Datenbank ist) hinzugezogen wird.
+
```
Comment (staudamm) [inline src/UserClassifier03.py]: This now empty comment block can be removed, no?

File: src/Categories.py [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+CAT2VAL = {}
+
+def p_dict(dict):
+    for i in dict.keys():
+        print('"{}" : {},\n'.format(i, dict[i]))
+
+for file in os.listdir('../TrainingDataNew'):
+    if not file.startswith('.') and not file.endswith('pkl'):
+        CATEGORY_NAMES.append(os.path.splitext(file)[0])
+        CATEGORY_NAMES.sort()
```
[the remark this answers is not in the document]
Comment (Regexose) [reply src/Categories.py]: done

File: src/Categories.py [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+        CAT2VAL = pickle.load(f)
+    CAT2VAL['Loesung'] = {'wheel': 2000, 'repeat': 2, 'cc_dict': normal_values, 'coarse': 37, 'fine': 0}
+    CAT2VAL['Gaga'] = {'wheel': 8000, 'repeat': 2, 'cc_dict': normal_values, 'coarse': 37, 'fine': 0}
+    #print('CAT2VAL nach pickle load: ', p_dict(CAT2VAL) )
+
+else:
+    CAT2VAL = dict_values(CATEGORY_NAMES, WHEEL, category_values, COARSE, FINE, REPEAT, )
+    #p_dict(CAT2VAL)
+    with open('../model_data/CAT2VAL.pkl', 'wb') as f:
+        pickle.dump(CAT2VAL, f, pickle.HIGHEST_PROTOCOL)
```
[the remark this answers is not in the document]
Comment (Regexose) [reply src/Categories.py]: done

File: src/Interpreter.py [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+        cat, prob = self.classifier.predict_proba(text, verbose=True)
+        print('input {}, cat {}, prob {}: '.format(text, cat, prob))
+        velo = CAT2VAL[cat]['velo']
+        pitch = CAT2VAL[cat]['pitch']
+        return [pitch, velo, cat, prob[0]]
+
+    def text2wheel(self, text):
+        cat, prob = self.classifier.predict_proba(text, filter_stop_words=False, verbose=True)
+        print('input {}, cat {}, prob {}: '.format(text, cat, prob))
+        # pitch = category_to_note[classified_category_name]
```
[the remark this answers is not in the document]
Comment (Regexose) [reply src/Interpreter.py]: yup

File: src/Interpreter.py [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+    def text2pitch(self, text):
+        cat, prob = self.classifier.predict_proba(text, verbose=True)
+        print('input {}, cat {}, prob {}: '.format(text, cat, prob))
+        velo = CAT2VAL[cat]['velo']
+        pitch = CAT2VAL[cat]['pitch']
+        return [pitch, velo, cat, prob[0]]
+
+    def text2wheel(self, text):
+        cat, prob = self.classifier.predict_proba(text, filter_stop_words=False, verbose=True)
+        print('input {}, cat {}, prob {}: '.format(text, cat, prob))
```
[the remark this answers is not in the document]
Comment (Regexose) [reply src/Interpreter.py]: yup

File: src/UDPClient.py (hunk 4 above)
Comment (Regexose) [reply src/UDPClient.py]: yup

File: src/UDPClient.py (hunk 5 above)
Comment (Regexose) [reply src/UDPClient.py]: yup

File: src/UDPClient.py (hunk 6 above)
Comment (Regexose) [reply src/UDPClient.py]: yup

File: src/UDPClient.py (hunk 7 above)
Comment (Regexose) [reply src/UDPClient.py]: yup

File: src/UserClassifier03.py (hunk 8 above)
Comment (Regexose) [reply src/UserClassifier03.py]: yup

**Comment by Regexose** [comment]
@staudamm 
habe ihren Anweisungen folge geleistet. Zudem habe ich die Erzeugung von .txt files unterbunden.
muss das Ganze nun gemerged werden, bevor ich den Classifiere implementiere?

**Comment by staudamm** [comment]
@staudamm 👍 , I have merge the branch into the master. Please checkout the master, pull, and then create a new branch for the next feature

**Pull request merged**